streams/src/main/java/org/apache/kafka/streams/processor/internals/assignment/AssignmentInfo.java

```diff
@@ -37,13 +37,13 @@
 import java.util.Map;
 import java.util.Set;
 
+import static org.apache.kafka.streams.processor.internals.assignment.StreamsAssignmentProtocolVersions.LATEST_SUPPORTED_VERSION;
+import static org.apache.kafka.streams.processor.internals.assignment.StreamsAssignmentProtocolVersions.UNKNOWN;
+
 public class AssignmentInfo {
 
     private static final Logger log = LoggerFactory.getLogger(AssignmentInfo.class);
 
-    public static final int LATEST_SUPPORTED_VERSION = 4;
-    static final int UNKNOWN = -1;
```

Comment:
These (and the SubscriptionInfo) are logically the same as the "assignment protocol version", so I've migrated them to `StreamsAssignmentProtocolVersions`.